tests/render/test_inline_renderer.py

```diff
@@ -9,11 +9,29 @@
     ExpectationValidationResult,
 )
 from great_expectations.data_context import DataContext
+from great_expectations.render.exceptions import InvalidRenderedContentError
 from great_expectations.render.renderer.inline_renderer import InlineRenderer
 from great_expectations.render.types import RenderedAtomicContent
 from great_expectations.validator.validator import Validator
 
 
+@pytest.mark.unit
+def test_inline_renderer_error_message(
+    alice_columnar_table_single_batch_context: DataContext,
+):
+    context: DataContext = alice_columnar_table_single_batch_context
+    expectation_suite: ExpectationSuite = context.create_expectation_suite(
+        "validating_alice_data"
+    )
+    with pytest.raises(InvalidRenderedContentError) as e:
+        InlineRenderer(render_object=expectation_suite)  # type: ignore
+    assert (
+        str(e.value)
+        == "InlineRenderer can only be used with an ExpectationConfiguration or ExpectationValidationResult, but <class 'great_expectations.core.expectation_suite.ExpectationSuite'> was used."
+    )
+
+
+@pytest.mark.unit
```

Comment:
Same note about `unit` here - this feels quite complex for a unit test and uses fixtures like `alice_context` which are quite cumbersome and have moving parts.

Reply:
I agree, this marker was a mistake. ✅